Sources/Fluent/Entity/Entity.swift

```diff
@@ -82,6 +84,10 @@ extension Entity {
     public static var name: String {
         return String(describing: self).lowercased()
     }
+    
+    public static var idType: Schema.Field.KeyType? {
```

Comment:
Ok, agreed. I suppose my general approach would be to fail early and decisively (preconditionFailure), but if a client of this API is attempting to for example create a schema with no database associated with the entity, they're going to find out they're doing it wrong also through some other mechanism than a precondition failure here.